bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/device/CommandHandler.java

```diff
@@ -386,13 +386,13 @@ public void handleCommand(InsteonChannelConfiguration conf, Command cmd, Insteon
             try {
                 int button = this.getIntParameter("button", -1);
                 if (cmd == OnOffType.ON) {
-                    Msg m = dev.makeExtendedMessage((byte) 0x1f, (byte) 0x2e, (byte) 0x00,
-                            new byte[] { (byte) button, (byte) 0x09, (byte) 0x01 });
+                    //Msg m = dev.makeExtendedMessage((byte) 0x1f, (byte) 0x2e, (byte) 0x00, new byte[] { (byte) button, (byte) 0x09, (byte) 0x01 });
+                    Msg m = dev.makeExtendedMessage((byte)31, (byte)32, (byte)9, new byte[3]);
                     dev.enqueueMessage(m, feature);
                     logger.debug("{}: sent msg to switch {} on", nm(), dev.getAddress());
                 } else if (cmd == OnOffType.OFF) {
-                    Msg m = dev.makeExtendedMessage((byte) 0x1f, (byte) 0x2e, (byte) 0x00,
-                            new byte[] { (byte) button, (byte) 0x09, (byte) 0x00 });
+                    //Msg m = dev.makeExtendedMessage((byte) 0x1f, (byte) 0x2e, (byte) 0x00, new byte[] { (byte) button, (byte) 0x09, (byte) 0x00 });
+                    Msg m = dev.makeExtendedMessage((byte)31, (byte)32, (byte)8, new byte[3]);
```

Comment:
Remove commented code.
Convert to hexadecimal, and explicitly assign values to the 3 bytes (even if they are 0x00).

Reply:
Done